src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-topic-list/topic.model.ts

```diff
@@ -0,0 +1,42 @@
+interface TopicDest {
+  push_endpoint: string;
+  push_endpoint_args: string;
+  push_endpoint_topic: string;
+  stored_secret: string;
+  persistent: boolean;
+  persistent_queue: string;
+  time_to_live: number;
+  max_retries: number;
+  retry_sleep_duration: number;
+}
+
+export interface TopicModel {
+  owner: string;
+  name: string;
+  arn: string;
+  dest: TopicDest;
+  opaqueData: string;
+  policy: string | {};
+  subscribed_buckets: any[];
+}
+
+export interface ApiResponse {
+  topics?: TopicModel[];
+}
+
+export const ENDPOINT_TYPE = {
+  HTTP: 'HTTP',
+  AMQP: 'AMQP',
+  Kafka: 'KAFKA'
+};
+export const AMQP_ACK_LEVEL  = {
+  Select: 'Select AMQP level',
```

Comment:
```suggestion
  select: 'Select AMQP level',
```
Also where is this used ?

Reply:
Suggested changes are done

Reply:
Suggested changes are done